NEWS.md

```diff
@@ -19,10 +19,16 @@
   sessions.
 - Add `predict` method to obtain conditional mean estimates and prediction intervals, both with or without previous observations.
 - Use automatic differentiation to calculate Satterthwaite adjusted degrees of freedom calculation, resulting in 10-fold speed-up of the Satterthwaite calculations after the initial model fit.
+- Two new methods added for mmrm models: `model.matrix()` and `terms()`.
```

Comment:
```suggestion
- Add methods `model.matrix()` and `terms()`.
```
you might also consider adding some more details if it makes sense, compare the other "add ... method" entries above

Reply:
updated with your suggested text and a bit of detail